tests/pyanaconda_tests/network_test.py

```diff
@@ -280,7 +280,12 @@ def setUp(self):
         self.ksdata_mock = mock.Mock()
         self.ksdata_mock.network = mock.Mock()
 
-    def update_hostname_data_test(self):
+    @patch('pyanaconda.dbus.DBus.get_proxy')
+    def update_hostname_data_test(self, proxy_getter):
+        proxy = mock.Mock()
+        proxy.SetHostname = mock.Mock()
```

Comment:
I am not sure if you need to create a mock for `SetHostname` as well. It should work without it.